projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/create.py

```diff
@@ -30,7 +30,13 @@ def __init__(self, rest_api_url: str):
 
     @ApiClientErrorDecorator()
     def create_job(
-        self, name: str, team: str, path: str, cloud: bool, local: bool
+        self,
+        name: str,
+        team: str,
+        path: str,
+        cloud: bool,
+        local: bool,
+        sample_job=None,
```

Comment:
use consistent argument names across methods if the semantics are the same. And use different if they have changed 

 . I got confused here . I thought sample_job_module was something different.

Reply:
I've fixed this, I noticed it after pushing

Reply:
resolving this since it was merged in a separate PR